sycl-fusion/passes/kernel-fusion/SYCLSpecConstMaterializer.cpp

```diff
@@ -229,6 +230,7 @@ void SYCLSpecConstMaterializer::reportAndReset() {
     }
   }
   LLVM_DEBUG(dbgs() << "\n\n");
+#endif
```

Comment:
```suggestion
    }
  }
  dbgs() << "\n\n";
});
```
Removing all `LLVM_DEBUG` and wrapping this in a `LLVM_DEBUG` block should fix too, right?

Reply:
I suspect so. Let's give it a shot!